nurseconnect/tests/test_profiles.py

```diff
@@ -104,6 +98,32 @@ def test_register_user_validation(self):
             [u"Username already exists."]
         )
 
+    def test_invalid_clinic_code_raises_error(self):
```

Comment:
This defies unit testing. What you are doing here is actually hitting/testing the API. This will get us into trouble if the API is down, or not functioning properly. It will probably also slow down tests.

What we should be doing is is mocking it. 
In this particular case, it shouldn't be that hard... but probably the methods that actually call the API will need some refactoring.